pages/common/git-range-diff.md

```diff
@@ -0,0 +1,12 @@
+# git range-diff
+
+> Compare two commit ranges (e.g. two versions of a branch).
+> More information: <https://git-scm.com/docs/git-range-diff>.
+
+- Diff the diffs of commit1 and commit2:
+
+`git range-diff {{commit1}}~..{{commit1} {{commit2}}~..{{commit2}}`
+
+- Diff the changes of ours and theirs from their common ancestor:
```

Comment:
I'm not sure what this command does, but I think there should be better words for `ours` and `theirs`. @bl-ue what do you think?

Reply:
Please look at the SO question I linked for a longer explanation

Reply:
Personally, I don't mind the usage of `ours` and `theirs` because that's official Git wording, and [we allow such exceptions](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#inclusive-language).

Reply:
However, we don't seem to use this terminology in any of the other Git pages, so maybe we should try to move away from it.  @xerus2000 also see the [`git-diff` page](https://github.com/tldr-pages/tldr/blob/a99a8d98d86e857ab1174804cfbecf8b170f9cd7/pages/common/git-diff.md).

Reply:
yep, I had a look at that.
tbh, if we want to be correct, the mentions of commit and branch in such pages should actually be replaced by `commit-ish` (see https://stackoverflow.com/a/23303550) since these are interchangeable.

Reply:
Wow, you're right. We might need to change that.  Let's wait for @sbrl's opinion then.

Reply:
Heya! Sorry for the wait here. I would suggest that `ours` and `theirs` would be best used when disambiguating merges - e.g. "our commits" and "their commits". I'm not sure it really helps clarity here - I suggest changing the wording here would be the best course of action to be more specific - both as to _what_ is being referred to, and also how their are different from one another.

The man page is also singularly unhelpful:

```
       <rev1>...<rev2>
           Equivalent to passing <rev2>..<rev1> and <rev1>..<rev2>.
```

I'm unsure as how useful this example is. In theory it looks like to me that this command would be most useful when used to view the differences between 2 slightly different patches.

For example, Bob creates a patch set for project Foo to add a new feature. Bob sends the patch set to Alice, who makes some changes to the patch set with `git rebase -i` and sends them back to Bob. Bob then uses `git range-diff` to compare Alice's patch set with their own local patch set.

Reply:
Yes, and checking the diff after a rebase is a core use-case of this command imo, that's why I want this example. But that still leaves the question of the wording open ;)

Reply:
Oh, I see! I'm not sure I totally understand how that would work though, so I'm unsure if I'm the best person to come up with wordingg here @xerus2000. Given that's the central use-case of this command though, it might make sense to base the example around that to make it more relatable and useful. Perhaps we could reword this example to talk specifically about checking a diff after a rebase?

Reply:
Tried adding examples :)